### PR TITLE
Verification and ssl path updated

### PR DESCRIPTION
**Pull request by canic-eiq**

### CODE REVIEW

File: extensions/group_ib/incoming_feeds/group-ib-attacks-phishing-kit/manifest.json [hunk 1]
```diff
@@ -47,20 +47,20 @@
           "name": "email"
         },
         {
-          "default": true,
-          "required": true,
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group-ib-attacks-phishing-kit/manifest.json:50]: Default value for "verify" should always be true.

File: extensions/crowdstrike/enrichments/crowdstrike_enricher/manifest.json [hunk 2]
```diff
@@ -57,28 +57,28 @@
...
+          "title": "SSL verification",
           "name": "verify"
         },
         {
           "default": "",
           "required": false,
-          "type": "string",
-          "description": "Comma separated list of ignored indicator relation values.",
-          "title": "Blacklisted relation values",
-          "name": "blacklist"
```
Comment (petarc-eiq) [inline extensions/crowdstrike/enrichments/crowdstrike_enricher/manifest.json:81]: Be careful what you're replacing, you removed "blacklist" field and have 2 "verify" fields.

File: extensions/group_ib/incoming_feeds/group-ib-brand-abuse-phishing-kit/manifest.json [hunk 3]
```diff
@@ -47,20 +47,20 @@
           "name": "email"
         },
         {
-          "default": true,
-          "required": true,
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group-ib-brand-abuse-phishing-kit/manifest.json:50]: Default value for "verify" should always be true.

File: extensions/group_ib/incoming_feeds/group-ib-compromised-data-accounts/manifest.json [hunk 4]
```diff
@@ -47,20 +47,20 @@
           "name": "email"
         },
         {
-          "default": true,
-          "required": true,
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group-ib-compromised-data-accounts/manifest.json:50]: Default value for "verify" should always be true.

File: extensions/group_ib/incoming_feeds/group-ib-compromised-data-cards/manifest.json [hunk 5]
```diff
@@ -47,20 +47,20 @@
           "name": "email"
         },
         {
-          "default": true,
-          "required": true,
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group-ib-compromised-data-cards/manifest.json:50]: Default value for "verify" should always be true.

File: extensions/group_ib/incoming_feeds/group-ib-human-intelligence/manifest.json [hunk 6]
```diff
@@ -47,20 +47,20 @@
           "name": "email"
         },
         {
-          "default": true,
-          "required": true,
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group-ib-human-intelligence/manifest.json:50]: Default value for "verify" should always be true. (apply everywhere... :) )

File: extensions/group_ib/incoming_feeds/group_ib_attacks_phishing/manifest.json [hunk 7]
```diff
@@ -47,21 +47,21 @@
...
           "required": false,
           "type": "file",
-          "description": "Path to SSL certificate file",
-          "title": "Path to SSL certificate file",
-          "name": "verify"
-        }
+          "description": "Insert file path to your SSL certificate here.",
+          "title": "Path to SSL certificate.",
+          "name": "custom_ssl"
+        }s
```
Comment (petarc-eiq) [inline extensions/group_ib/incoming_feeds/group_ib_attacks_phishing/manifest.json:64]: Delete "s".

File: extensions/intel471/enrichments/adversary-intelligence-enricher/manifest.json [hunk 8]
```diff
@@ -42,17 +46,30 @@
...
-          "default": "***@***.****",
+          "default": "",
           "required": true,
           "type": "string",
           "description": "A valid email address to be granted access to the Intel 471 API endpoint.",
           "title": "email",
           "name": "email"
+        },
+        {
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/intel471/enrichments/adversary-intelligence-enricher/manifest.json:57]: Default value for "verify" should always be true.

File: extensions/intel471/enrichments/posts_enricher/manifest.json [hunk 9]
```diff
@@ -55,6 +55,22 @@
           "description": "Place value here to lookup forum posts the actor has written.",
           "title": "Actor",
           "name": "actor"
+        },
+       {
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/intel471/enrichments/posts_enricher/manifest.json:60]: Default value for "verify" should always be true.

File: extensions/intel471/enrichments/posts_enricher/utils.py [hunk 10]
```diff
@@ -47,7 +47,7 @@ def fetch_with_paging(
     while True:
         query_params.update({"count": page_size, "offset": offset})
         response = fetch_results(
-            self, url=api_url, auth=auth, verify_ssl=verify_ssl, params=query_params
+            self, url=api_url, auth=auth, verify=verify_ssl, params=query_params
```
Comment (petarc-eiq) [inline extensions/intel471/enrichments/posts_enricher/utils.py:50]: fetch_results parameter is `verify_ssl`, undo this change.

File: extensions/intel471/incoming_feeds/adversary_Intelligence_feed/main.py [hunk 11]
```diff
@@ -53,7 +54,7 @@ def download(self):
                     furl(api_url).add(path=REPORT_ENDPOINT.format(report["uid"])).url
                 )
                 detailed_report = fetch_results(
-                    self, report_url, auth=(api_email, api_key), verify_ssl=verify
+                    self, report_url, auth=(api_email, api_key), verify=verify_ssl
```
Comment (petarc-eiq) [inline extensions/intel471/incoming_feeds/adversary_Intelligence_feed/main.py:57]: fetch_results parameter is `verify_ssl`, so it should be `verify_ssl=verify_ssl`.

File: extensions/intel471/incoming_feeds/adversary_Intelligence_feed/manifest.json [hunk 12]
```diff
@@ -37,12 +37,28 @@
...
-          "default": "***@***.****",
+          "default": "",
           "required": true,
           "type": "string",
           "description": "email",
           "title": "email",
           "name": "email"
+        },
+        {
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/intel471/incoming_feeds/adversary_Intelligence_feed/manifest.json:48]: `"default": true,`

File: extensions/intel471/incoming_feeds/malware_feed/manifest.json [hunk 13]
```diff
@@ -37,12 +39,28 @@
...
+          "default": "",
           "required": true,
           "type": "string",
           "description": "email",
           "title": "email",
           "name": "email"
+        },
+        {
+          "default": false,
+          "required": false,
```
Comment (petarc-eiq) [inline extensions/intel471/incoming_feeds/malware_feed/manifest.json:51]: `"default": true,`

File: extensions/intel471/incoming_feeds/vulnerability_intelligence/manifest.json [hunk 14]
```diff
@@ -37,12 +37,28 @@
...
-          "default": "***@***.****",
+          "default": "*",
           "required": true,
           "type": "string",
           "description": "email",
           "title": "email",
           "name": "email"
+        },
+        {
+          "default": false,
```
Comment (petarc-eiq) [inline extensions/intel471/incoming_feeds/vulnerability_intelligence/manifest.json:48]: `"default": true,`

**Review by petarc-eiq — CHANGES_REQUESTED** [review]
small typos and such

File: extensions/crowdstrike/enrichments/crowdstrike_enricher/manifest.json [hunk 15]
```diff
@@ -57,20 +57,28 @@
...
-          "name": "custom_ssl"
+          "name": "verify"
+        },
+        {
+          "default": true,
+          "required": false,
+          "type": "boolean",
+          "description": "Select to enable SSL verification.",
+          "title": "SSL verification",
+          "name": "verify"
```
Comment (petarc-eiq) [inline extensions/crowdstrike/enrichments/crowdstrike_enricher/manifest.json:73]: Now there are 2 "verify" parameters. Still not good.

File: extensions/flashpoint/incoming_feeds/flashpoint_reports_provider/manifest.json [hunk 16]
```diff
@@ -55,7 +55,7 @@
           "name": "custom_ssl"
         },
         {
-          "default": false,
+          "default": true,
```
Comment (petarc-eiq) [inline extensions/flashpoint/incoming_feeds/flashpoint_reports_provider/manifest.json:58]: Did you mean to change this? This is cyber_threats, not verify :)

**Review by petarc-eiq — CHANGES_REQUESTED** [review]

**Review by petarc-eiq — APPROVED** [review]

File: extensions/intel471/enrichments/posts_enricher/utils.py (hunk 10 above)
Comment (erwin-eiq) [reply extensions/intel471/enrichments/posts_enricher/utils.py:50]: @canic-eiq friendly ping on this comment :)

File: extensions/intel471/incoming_feeds/adversary_Intelligence_feed/main.py (hunk 11 above)
Comment (erwin-eiq) [reply extensions/intel471/incoming_feeds/adversary_Intelligence_feed/main.py:57]: @canic-eiq friendly ping on this comment :)

**Pull request merged**